pgsql-flexible-server.tf

```diff
@@ -151,8 +151,9 @@ resource "null_resource" "set-user-permissions-additionaldbs" {
     command = "${path.module}/set-postgres-permissions.bash"
 
     environment = {
-      DB_HOST_NAME   = azurerm_postgresql_flexible_server.pgsql_server.fqdn
+      PGHOST         = azurerm_postgresql_flexible_server.pgsql_server.fqdn
```

Comment:
changing to using `psql` variable names so we don't have to repeat them all the time and it makes the cli shorter and script easier to understand